Installation/include/CGAL/config.h

```diff
@@ -728,4 +728,56 @@ typedef const void * Nullptr_t;   // Anticipate C++0x's std::nullptr_t
 /// @}
 #include <CGAL/license/lgpl.h>
 
+//----------------------------------------------------------------------//
+//  Function to define data directory
+//----------------------------------------------------------------------//
+#include <cstdlib>
+#include <string>
+#include <fstream>
+#include <iostream>
+
+namespace CGAL {
+
+// Returns filename prefixed by the directory of CGAL containing data.
+// This directory is either defined in the environement variable CGAL_DATA_DIR,
+// otherwise it is taken from the constant CGAL_DATA_DIR (defined in CMake),
+// otherwise it is empty (and thus returns filename unmodified).
+inline std::string get_data_file_path(const char* filename)
+{
+ std::string res;
+
+ const char *cgal_dir=std::getenv("CGAL_DATA_DIR");
+
+ if (cgal_dir!=nullptr)
+ { res=std::string(cgal_dir); }
+#ifdef CGAL_DATA_DIR
+ else
+ { res=std::string(CGAL_DATA_DIR); }
+#endif
```

Comment:
I have the impression that the code could be simplified:
```suggestion
 const char *cgal_dir=std::getenv("CGAL_DATA_DIR");

#ifdef CGAL_DATA_DIR
 if (cgal_dir==nullptr)
 { cgal_dir=CGAL_DATA_DIR; }
#endif
```
Because, if it is defined, `CGAL_DATA_DIR` is supposed to evaluate to a string literal.

Do not be surprised: in
```c++
  const char *cgal_dir=...
```
`cgal_dir` is not a constant pointer, but a pointer to a constant char array, and it can be reassigned.

Reply:
Thanks @lrineau ; but this is not enough. I need to copy cgal_dir into res, and before the copy I need to test if cgal_dir is nullptr (because I think it is an error to construct a string from nullptr). Thus not sure your suggestion simplify the code...

Reply:
I now it is not enough. That code was incomplete. I wanted to show how to avoid testing `CGAL_DATA_DIR` several times.